fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -384,15 +388,26 @@ struct TenantManagementWorkload : TestWorkload {
 		} else {
 			ASSERT_EQ(operationType, OperationType::METACLUSTER);
 			ASSERT_EQ(tenantsToCreate.size(), 1);
-			metacluster::MetaclusterTenantMapEntry entry =
-			    metacluster::MetaclusterTenantMapEntry::fromTenantMapEntry(tenantsToCreate.begin()->second);
-			auto assign = metacluster::AssignClusterAutomatically::True;
+			entry = metacluster::MetaclusterTenantMapEntry::fromTenantMapEntry(tenantsToCreate.begin()->second);
 			if (deterministicRandom()->coinflip()) {
 				entry.assignedCluster = self->dataClusterName;
 				assign = metacluster::AssignClusterAutomatically::False;
 			}
 
-			wait(metacluster::createTenant(self->mvDb, entry, assign));
+			try {
+				wait(metacluster::createTenant(self->mvDb, entry, assign, ignoreCapacityLimit));
+				ASSERT(!assign || !ignoreCapacityLimit);
+			} catch (Error& e) {
+				if (e.code() == error_code_invalid_tenant_configuration) {
+					if (!entry.tenantGroup.present()) {
+						ASSERT(assign && ignoreCapacityLimit);
+					} else {
+						// TODO: to verify the assigned cluster is different from the tenant group's current assigned
+						// cluster, we need to track this info for each tenant group.
```

Comment:
There is only one data cluster and we always choose it as the assigned location, so I don't think this case is possible

Reply:
Done